README.md

```diff
@@ -58,6 +58,21 @@ yarn start --sources 'packages/esm-patient-biometrics-app' --sources 'packages/e
 
 Alternatively, you could run `yarn serve` from within the individual packages and then use [import map overrides](http://o3-dev.docs.openmrs.org/#/getting_started/setup?id=import-map-overrides).
 
+## Troubleshooting
+
+If you notice that your local version of the application is not working or that there's a mismatch between what you see locally versus what's in the reference application, you likely have outdated versions of core libraries. To update core libraries, run the following commands:
```

Comment:
Does `in the reference application` mean `dev3.openmrs.org`?

Reply:
Correct.

Reply:
Assuming `demo` site does not mean `https://demo.openmrs.org/` 😊

Reply:
I guess dev3 would be the safest bet then :)

Reply:
How can i say NO? 😊

Reply:
Is that a yes?

Reply:
Loool, yes. 🤣